documentation/intermediate_ca.md

```diff
@@ -31,39 +31,45 @@ The following diagram shows the configuration of Puppet's certificate infrastruc
 To set up Puppet as an intermediate CA with an external root:
 
 1. Collect your organization’s chain of trust. This includes:
-* The root cert
-* Any intermediate CA certs
-* The CA cert that you will use to issue a CA signing cert for your Puppet infrastructure
+   * The root cert
+   * Any intermediate CA certs
+   * The CA cert that you will use to issue a CA signing cert for your Puppet infrastructure
 2. Collect the corresponding CRLs for each of these certificates. Take note of the expiration dates of each CRL. When one expires, you need to refresh it for Puppet to continue working.
 3. Create a private key for the Puppet CA — take note of this, you will need to import it into your Puppet infrastructure later.
 4. Create a CSR for the Puppet CA and sign it using the appropriate cert from your organization’s trust chain, which you gathered in Step 1. This is the new Puppet CA cert, which will be used to sign all other Puppet infrastructure certs.
 5. Create a CRL for the new Puppet CA cert.
 6. Concatenate all of the certs into a PEM file, starting with the new Puppet CA cert and ending with your organization’s root cert. The file should contain the PEM-encoded certs, like this:
```

Comment:
nitpick, but do we want to change all these to `1.`, for consistency and future ease-of-rearrangement?

Reply:
Okay, sounds good, I'll go back and change it.

Reply:
fixed and updated the PR